### PR TITLE
[ci-app] JUnit XML Upload – Use Safe File Names

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What and why?

Use a function to make sure the file name we use to be stored in S3 is safe.

Also, improve rendered message when using a single file.

### How?

* Create `getSafeFileName`. 
* Render a slightly different name if the input is a single file.

### Review checklist

- [x] Feature or bugfix MUST have appropriate tests (unit, integration)

### CODE REVIEW

File: src/commands/junit/api.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ import {renderUpload} from './renderer'
 
 import {CI_JOB_URL, CI_PIPELINE_URL, GIT_SHA} from '../../helpers/tags'
 
+const getSafeFileName = (unsafeFileName: string) => unsafeFileName.replace(/[^a-z0-9]/gi, '_')
```
Comment (drodriguezhdez) [inline src/commands/junit/api.ts:13]: Can we have a test of this function? Also, it'd worth it to have a comment explaining why this change in the name is required. (Are there some requirements in the name of the file for S3)?

Comment (juan-fernandez) [reply src/commands/junit/api.ts]: done

**Review by drodriguezhdez — APPROVED** [review]
